setup.py

```diff
@@ -30,41 +30,49 @@
     "hyperopt>=0.1.2",
 ]
 
-setup_requirements = ['pytest-runner', ]
+setup_requirements = ["pytest-runner"]
 
-test_requirements = ['pytest', ]
+test_requirements = ["pytest"]
 
-extras_requirements = {'test': ['scanpy', 'louvain', "leidenalg>=0.7.0", "python-igraph>=0.7.1", ]}
+extras_requirements = {"test": ["scanpy", "louvain", "leidenalg>=0.7.0", "python-igraph>=0.7.1", ]}
 
-author = 'Romain Lopez, Jeffrey Regier, Maxime Langevin, Edouard Mehlman, Yining Liu'
+author = (
+    "Romain Lopez, "
+    "Jeffrey Regier, "
+    "Maxime Langevin, "
+    "Edouard Mehlman, "
+    "Yining Liu, "
+    "Achille Nazaret, "
+    "Gabriel Misrachi"
```

Comment:
Would you add Pierre and Oscar too please?

Reply:
of course